api/v1alpha1/tls_types.go

```diff
@@ -110,6 +110,12 @@ const (
 // to the Gateway.
 // By default, no client specific configuration is validated.
 type ClientValidationContext struct {
+	// RequiredClientCertificate set to true means that Envoy will reject connections without a valid client certificate.
```

Comment:
I'd flip the meaning of the noun here, so the default value is `false`
The two terms that come to mind are `optional` and `allowMissing`
https://github.com/envoyproxy/gateway/pull/3157 is trying to address a similar problem for JWT

Reply:
we cannot set the default value to false to support backward compatibility.
should we use `DisabledClientCertificateValidation`?

Reply:
thats too wordy imo

Reply:
since it's under ClientValidation, should we set this as Enabled or Disabled? hmm

Reply:
the meaning of the noun picked, should convey the default meaning, e.g. `optional` or `allowMissing`  highlights that the default is `false`

Reply:
if that is the case, I propose with `Required` and the default is true.
since it's under ClientValidationContext struct.

or do you have better name @arkodg?

Reply:
I prefer `optional` which defaults to `false`
will bring this up in the community meeting this week to speed up consensus on this

Reply:
I am using `Required` and set default to `true`.
I am not sure why I cannot use `!` whn using optional. somehow translator_test.go failing

Reply:
Both names sound good to me. However, I prefer to use `false` as default to keep consistent with other APIs.

Reply:
Hi @arkodg @zhaohuabing change `Required` to `Optional` and use false by default.
please help recheck! thanks